src/hotspot/share/gc/shenandoah/shenandoahFreeSet.cpp

```diff
@@ -112,15 +115,21 @@ HeapWord* ShenandoahFreeSet::allocate_single(ShenandoahAllocRequest& req, bool&
       break;
     }
     case ShenandoahAllocRequest::_alloc_gclab:
+      // GCLABs are for evacuation so we must be in evacuation phase.  If this allocation is successful, increment
+      // the relevant evac_expended rather than used value.
+      allocating_gclab = true;
```

Comment:
Do we really need a new parameter here? The information is carried in the type of the allocation request (`req`) which is also passed to `allocate_with_affiliation` (the request goes all the way down to `try_allocate_in`).

Reply:
Thank you.  I will change this.